src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -414,19 +432,27 @@ private void setupHosting(Channel channel, User sender, String hostingId, String
                 return;
             }
 
+            String prUrl = createUploadPermissionPR(channel,sender,issueID,forkTo,users,Collections.singletonList(defaultAssignee));
+            if(StringUtils.isBlank(prUrl)) {
+                out.message("Could not create upload permission pull request");
+            }
+
             // update the issue with information on next steps
             String msg = "At the request of " + sender.getNick() + " on IRC, "
                     + "the code has been forked into the jenkinsci project on GitHub as "
                     + "https://github.com/jenkinsci/" + forkTo
                     + "\n\nA JIRA component named " + forkTo + " has also been created with "
                     + defaultAssignee + " as the default assignee for issues."
-                    + "\n\nPlease remove your original repository so that the jenkinsci repository "
+                    + "\n\nA [pull request|" + prUrl +"] has been created against the repository permissions updater to "
+                    + "setup release permissions for " + defaultAssignee + ". Additional users can be added by modifying "
+                    + "the created file. " + defaultAssignee + " will need to login to Jenkins' [Artifactory|https://repo.jenkins-ci.org/webapp/#/login] once before the permissions will be merged."
+                    + "\n\nPlease remove your original repository so that the jenkinsci organization repository "
```

Comment:
While we're here, could we update this message to say that deleting only helps when there are no other forks (optionally: otherwise request they contact GH support?)

Reply:
Please check the updated message and let me know if it says what you want it to say.